### PR TITLE
Add ut for ShadowExtractor in shadow project 

### PR DESCRIPTION
**Pull request by zzs52**
Fixes Add ut for ShadowExtractor in shadow project  #12498.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class ShadowExtractorTest {
```
Comment (soulasuna) [inline shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java:31]: Please use final.

File: shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Test
+    public void extractValuesIsSimpleExpressionSegment() {
+        LiteralExpressionSegment LiteralExpressionSegment = new LiteralExpressionSegment(1, 2, "");
+        List<Object> list = new LinkedList<>();
+        Collection<Comparable<?>> result = new LinkedList<>();
+        result.add((Comparable<?>) LiteralExpressionSegment.getLiterals());
+        assertThat(ShadowExtractor.extractValues(LiteralExpressionSegment, list), equalTo(Optional.of(result)));
```
Comment (soulasuna) [inline shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java:39]: 使用 `assertThat` 规范 请参考下文中 `assertThat(operationType, is(DistributedTransactionOperationType.IGNORE));`

**Review by soulasuna — COMMENTED** [review]
Thank you very much for your contribution.
There are two minor problems, please fix.

File: shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java (hunk 1 above)
Comment (zzs52) [reply shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java]: Ok,thanks.

File: shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java (hunk 2 above)
Comment (zzs52) [reply shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/test/java/org/apache/shardingsphere/shadow/route/engine/util/ShadowExtractorTest.java]: Ok,thanks.

**Pull request closed without merging**